src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = 40;
+
+  let daysCost = sum * days;
```

Comment:
daysCost is also not the best name. Customer pays not just for days, but for some service. Something like rentalCost might sound better